app/controllers/users_controller.rb

```diff
@@ -336,9 +336,11 @@ def destroy_request_in_progress?
   def import_articles_from_feed(user)
     return if user.feed_url.blank?
 
-    worker = FeatureFlag.enabled?(:feeds_import) ? Feeds::ImportArticlesWorker : RssReaderFetchUserWorker
-
-    worker.perform_async(user.id)
+    if FeatureFlag.enabled?(:feeds_import)
```

Comment:
Do we still need this feature flag? Seems like we have had the`Feeds::ImportArticlesWorker` for a bit and it has been working.

Reply:
That's what I want to remove in the next PR.

The next PR will remove `RssReader`, feature flags and everything. Time to rip the bandaid off :D